src/pages/home/Home.tsx

```diff
@@ -24,71 +27,77 @@ function Home() {
         </tr>
       </thead>
       <tbody className="tbody">
-        {MOCKED_STUDENTS.map((student) => (
-          <tr>
-            <td></td>
-            <td>{student.dni}</td>
-            <td>{student.name}</td>
-            <td>{student.lastname}</td>
-            <td
-              style={{
-                color: student.status === "A término" ? "#00bf63" : "#ff3131",
-              }}
-            >
-              <strong> {student.status}</strong>
-            </td>
-            <td
-              style={{
-                color: student.isActive ? "#00bf63" : "#ff3131",
-              }}
-            >
-              <strong>{student.isActive ? "Activo" : "Inactivo"}</strong>
-            </td>
-            <td>
-              <div className="actions-container">
-                <button
-                  className="btn-payment"
-                  onClick={() => alert(`Registrar pago de ${student.name}`)}
+        {data &&
+          data.map((student) => (
```

Comment:
Y acá deberíamos modificarlo así


```suggestion
        {students &&
          students.map((student) => (
```